### PR TITLE
Added Navigation Component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Navigation Component

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Looks good with use of Utility classes

File: styles/navigation.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.c-navbar a {
+  color: var(--secondary-color);
+  text-decoration: none;
+}
+
+.c-navbar ul li {
+  display: inline-block;
+}
+
+.nav-input {
```
Comment (RakshithBhat03) [inline styles/navigation.css]: Just a small suggestion, maybe you can use `navbar-input` here. It makes it more consistent.

**Review by RakshithBhat03 — APPROVED** [review]
Suggested a small change. Otherwise, everything looks good. Good to merge.

File: styles/navigation.css (hunk 1 above)
Comment (nareshpoonia) [reply styles/navigation.css]: Thanks for the suggestion, changed the name

**Pull request merged**